prow/plugins.yaml

```diff
@@ -222,6 +222,15 @@ repo_milestone:
     maintainers_id: 2460384
     maintainers_team: kubernetes-milestone-maintainers
 
+repo_project:
+  # Default maintainer for projects
+  '':
```

Comment:
@stevekuznetsov for approach 2, does something like this work?
```
project:
  - orgs:
    - kubernetes:
      # Org level config, which can be overriden by repo level configs under repos
      maintainers_team_id: 2460384
      maintainers_team_name: kubernetes-milestone-maintainers
      columns:
        - To do
        - To triage
        - Backlog
      - repos:
        - kubernetes:
          maintainers_team_id: 2460384
          maintainers_team_name: kubernetes-milestone-maintainers
          columns:
            - To do
            - To triage
            - Backlog
    - kubeflow:
      maintainers_team: [""]
      columns:
        - To do
        - To triage
        - Backlog
```

Reply:
or something like this? i think this might be more readable:
```
project:
  # Org level config, which can be overriden by repo level configs under repos
  kubernetes/*:
  - maintainer_team_id: 2460384
    maintainers_team_name: kubernetes-milestone-maintainers
    columns:
    - To do
    - To triage
    - Backlog
  kubernetes/kubernetes:
  - maintainer_team_id: 2460384
    maintainers_team_name: kubernetes-milestone-maintainers
    columns:
    - To do
    - To triage
    - Backlog
```

Reply:
I think your first mockup is more true to the `branchprotection` config and I would rather see that one, but don't let me be the only opinion here!

Reply:
@stevekuznetsov I see. I prefer the second one because it's easier to read IMHO

Reply:
Totally.  @fejta thoughts? Another data point is (unfortunately) we have momentum behind the approach that already exists in our config, and keeping things similar would help everyone keep it straight

Reply:
@stevekuznetsov yeah I see. It makes sense to keep it consistent with existing configs